# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1304699948 25200
#      Fri May 06 09:39:08 2011 -0700
# Node ID b7f16997df4d2f623700c6779df86a60473c1dec
# Parent  8579e7a6e6e0c95c560fb46d56c435bbeae8e86a
maint: Move gmres.m to sparse directory rather than linear-algebra directory.

* sparse/module.mk: Update list to include gmres.m
* sparse/gmres.m: Newly moved function

diff --git a/scripts/linear-algebra/gmres.m b/scripts/sparse/gmres.m
copy from scripts/linear-algebra/gmres.m
copy to scripts/sparse/gmres.m
diff --git a/scripts/sparse/module.mk b/scripts/sparse/module.mk
--- a/scripts/sparse/module.mk
+++ b/scripts/sparse/module.mk
@@ -1,15 +1,16 @@
 FCN_FILE_DIRS += sparse
 
 sparse_FCN_FILES = \
   sparse/bicgstab.m \
   sparse/cgs.m \
   sparse/colperm.m \
   sparse/etreeplot.m \
+  sparse/gmres.m \
   sparse/gplot.m \
   sparse/nonzeros.m \
   sparse/pcg.m \
   sparse/pcr.m \
   sparse/spaugment.m \
   sparse/spconvert.m \
   sparse/spdiags.m \
   sparse/speye.m \
